### ISSUE TITLE
APTAKUBECONFIG environment variable locations for kubeconfig files not passed to pod shell-exec

### ISSUE THREAD

**Issue by gimbalgambit** [aptakube/aptakube#48]
### Current and expected behavior

If I use the (handy!) `APTAKUBECONFIG` environment variable to set the location of multiple clusters' context files, launching a pod shell terminal fails by passing an empty `KUBECONFIG= ` value to the shell exec command. 

It seems like loading all those contexts from those files probably needs to remember the map from context name to kubeconfig file path, so that the correct file path can be passed into that exec command, populating the `KUBECONFIG` value explicitly rather than implicitly.

### Additional context

If I swap to multiple files listed in the `KUBECONFIG` env var, things work fine... but all those contexts' paths are passed to the shell exec command. Due to some other CLI tooling risks, I don't want to keep all my contexts listed in `KUBECONFIG` at all times.

### Operating System

macOS Monterey v12.6.2, M1 Pro.

### Environment

```
clientVersion:
  buildDate: "2022-05-24T12:56:35Z"
  compiler: gc
  gitCommit: eae22ba6238096f5dec1ceb62766e97783f0ba2f
  gitTreeState: clean
  gitVersion: v1.22.10
  goVersion: go1.16.15
  major: "1"
  minor: "22"
  platform: darwin/arm64
```

**Comment by tobalsgithub** [comment]
One workaround you can use is to use the command line to start the app, setting `KUBECONFIG=$APTAKUBECONFIG`. 

```
KUBECONFIG=$APTAKUBECONFIG /Applications/Aptakube.app/Contents/MacOS/Aptakube
```

This leaves the app running in the foreground of the terminal, but then I can `ctrl+z` followed by `bg` and it seems to work perfectly. Even with multiple contexts selected it knows which one to use and connects appropriately. 

Would be nice to have this built in, but this workaround does the trick for me nicely. 

Really liking aptkube so far!

**Comment by goenning** [comment]
Thanks @tobalsgithub for sharing your workaround, happy to see it works, but it's not ideal! I'll get this one fixed as well.

**Comment by goenning** [comment]
This should be fixed on 1.1.0, but please reopen if it doesn't!
